wolox-books/src/app/screens/BooksList/components/books-list/books-list.component.ts

```diff
@@ -10,11 +11,14 @@ export class BooksListComponent {
 
   books: Book[] = [];
 
-  constructor() {
-    for (let i = 0; i < 8; i++) {
-      this.books.push(
-        { image: 'assets/book-cover.png', author: 'Título del libro', title: 'Autor del libro' }
-      );
-    }
+  constructor(private bookService: BookService) { }
+
+  ngOnInit(): void {
+    this.bookService.getBooks().subscribe(
+      res => {
+        this.books = res['page'].map(book => ({ image: book.image_url, author: book.author, title: book.title }));
```

Comment:
```suggestion
        this.books = res.page.map(book => ({ image: book.image_url, author: book.author, title: book.title }));
```

Reply:
No, I wanted to have it like that, but that's not posible, because as a call back, it doesn't know the inner properties of `res` so it makes a compilation error. I have to access it as brackets notation 😕